src/parser/sparqlParser/SparqlQleverVisitor.h

```diff
@@ -1172,7 +1172,7 @@ class SparqlQleverVisitor : public SparqlAutomaticVisitor {
       SparqlAutomaticParser::PnameLnContext* ctx) override {
     string text = ctx->getText();
     auto pos = text.find(':');
-    auto pnameNS = text.substr(0, pos + 1);
+    auto pnameNS = text.substr(0, pos);
```

Comment:
Was this a bug already before or is this specific for the CONSTRUCT clause?

Reply:
I wouldn't consider it a bug, but rather an implementation specific oddity.
Basically the prefix map was storing the trailing colon `:` along with the prefix name, but the non-antlr parser already stripped those colons from the prefix.
That's why previously the hash-map mapped `key:` to `value` instead of `key` to `value` and probably also the reason why the tests are currently failing